include/tcpdf_static.php

```diff
@@ -128,39 +128,6 @@ public static function getTCPDFProducer() {
 		return "\x54\x43\x50\x44\x46\x20".self::getTCPDFVersion()."\x20\x28\x68\x74\x74\x70\x3a\x2f\x2f\x77\x77\x77\x2e\x74\x63\x70\x64\x66\x2e\x6f\x72\x67\x29";
 	}
 
-	/**
-	 * Sets the current active configuration setting of magic_quotes_runtime (if the set_magic_quotes_runtime function exist)
-	 * @param boolean $mqr FALSE for off, TRUE for on.
-	 * @since 4.6.025 (2009-08-17)
-	 * @public static
-	 */
-	public static function set_mqr($mqr) {
```

Comment:
The magic quotes has nothing to do with phpstan, you should extract in in a commit like I had did: https://github.com/code-lts/WarnockPDF/commit/d0b14068e601e0738e4b465677b5cad0a5905a7c

Reply:
Slighly does, no support caught by phpstan ;-)

Anyway, discussed here: https://github.com/tecnickcom/TCPDF/pull/307#discussion_r602792776